certora/specs/SocialRecoveryModule.spec

```diff
@@ -40,6 +43,21 @@ function sumarizeSafeIsOwner(address callee, address owner) returns bool {
     return safeContract.isOwner(owner);
 }
 
+// A summary function that asserts that all `ISafe.getOwners` calls are done
+// to the `safeContract`, returning the same result as `safeContract.getOwners()`.
+function sumarizeSafeGetOwners(address callee) returns address[] {
+    assert callee == safeContract;
+    return safeContract.getOwners();
+}
+
+// A summary function that asserts that all `ISafe.execTransactionFromModule` calls are done
+// to the `safeContract`, and returns a bool based on the Social Recovery Module enable status.
+// Here we are only concerned if the module is enabled or not, and not the actual tx execution.
+function summarizeSafeExecTransactionFromModule(address callee) returns bool {
+    assert callee == safeContract;
+    return safeContract.isModuleEnabled(currentContract);
```

Comment:
Hmm, this does not have the same semantics as the Safe, namely it does not revert if `isModuleEnabled` is false.

Reply:
Jochen from Certora gave a better idea:

```
    if (callee == safeContract) {
        env e;
        assert safeContract.isModuleEnabled(currentContract);
        require safeContract.execTransactionFromModule(e, to, value, data, operation);
    }
    return _;
```

Will make this change soon, and see if it works 👍🏾